.eslintrc.json

```diff
@@ -29,7 +29,8 @@
       "rules": {
         "@typescript-eslint/no-empty-interface": 0,
         "@typescript-eslint/no-explicit-any": 0,
-        "@typescript-eslint/no-unused-vars": 0
+        "@typescript-eslint/no-unused-vars": 0,
+        "@typescript-eslint/ban-types": 0
```

Comment:
Why would you like to disable (`0`) the `ban-types` here?

Reply:
I'm curious to learn about the challenges you encountered that compelled you to switch from using `BigInt` to `bigint`. Since `BigInt` is a primitive type, it's often recommended to use `bigint` for typing, similar to how we use `string` instead of `String` and `number` instead of `Number`. Could you please share your experiences?

Reply:
Understood! 👍 The Graph Protocol has now also disabled [this rule](https://github.com/graphprotocol/graph-tooling/blob/main/.eslintrc.cjs#L16). Thank you for providing this information!